components/table/Table.tsx

```diff
@@ -205,12 +214,18 @@ function Table<RecordType extends object = any>(props: TableProps<RecordType>) {
     }
 
     if (onChange) {
-      onChange(changeInfo.pagination!, changeInfo.filters!, changeInfo.sorter!, {
-        currentDataSource: getFilterData(
-          getSortData(rawData, changeInfo.sorterStates!, childrenColumnName),
-          changeInfo.filterStates!,
-        ),
-      });
+      onChange(
+        changeInfo.pagination!,
+        changeInfo.filters!,
+        changeInfo.sorter!,
+        {
+          currentDataSource: getFilterData(
+            getSortData(rawData, changeInfo.sorterStates!, childrenColumnName),
+            changeInfo.filterStates!,
+          ),
+        },
```

Comment:
Please add action into 4th argument.

Reply:
I thought if someone is using the extra argument, moving its position can possible break people apps

Reply:
Yes, your statement is consistent with @afc163, but your code is not.

Reply:
This is for extra argument, to avoid long arguments.

```
onChange(pagination, filters, sorter, { currentDataSource, xxx, yyy, zzz })
```

instead of

```
onChange(pagination, filters, sorter, xxx, yyy, zzz)
```

Reply:
oh, got it